test/suite/test_bulk02.py

```diff
@@ -72,11 +71,8 @@ def test_bulkload_checkpoint(self):
         # In the case of named checkpoints, verify they're still there,
         # reflecting an empty file.
         if self.ckpt_type == 'named':
-            cursor = self.session.open_cursor(
-                self.uri, None, 'checkpoint=myckpt')
-            self.assertEquals(cursor.next(), wiredtiger.WT_NOTFOUND)
-            cursor.close()
-
+            self.assertRaises(wiredtiger.WiredTigerError,
+                lambda: self.session.open_cursor(self.uri, None, 'checkpoint=myckpt'))
```

Comment:
To be clear, as far as I know, this is the only functional change -- named checkpoints will skip files that are in the process of being bulk-loaded when the checkpoint starts.

OTOH, it's possible to create new tables while a checkpoint is in progress, which is a big win for some workloads.